echopype/visualize/plot.py

```diff
@@ -41,31 +39,31 @@ def _set_label(
 
         for idx, ax in enumerate(fg.axes.flat):
             name_dicts = fg.name_dicts.flatten()
-            if 'frequency' in name_dicts[idx]:
-                freq = name_dicts[idx]['frequency']
-                if col == 'frequency':
+            if 'channel' in name_dicts[idx]:
+                chan = name_dicts[idx]['channel']
+                if col == 'channel':
                     ax.set_title('')
             else:
-                freq = frequency
+                chan = channel
                 ax.set_title(f'Beam {fg.col_names[idx]}')
             ax.text(
                 *text_pos,
-                f"{int(freq / 1000)} kHz",
+                f"{chan} kHz",
```

Comment:
It seems that `chan` here is a string from our new coordinate `channel`? Before the indexing was based on frequency so "kHz" makes sense, but not with the current `channel`.